src/dialects/ibmi/query-generator.js

```diff
@@ -0,0 +1,836 @@
+'use strict';
+
+const Utils = require('../../utils');
+const util = require('util');
+const _ = require('lodash');
+const AbstractQueryGenerator = require('../abstract/query-generator');
+const DataTypes = require('../../data-types');
+const Model = require('../../model');
+const SqlString = require('../../sql-string');
+
+const typeWithoutDefault = new Set(['BLOB']);
+
+class IBMiQueryGenerator extends AbstractQueryGenerator {
+
+  // Version queries
+  versionQuery() {
+    return 'SELECT CONCAT(OS_VERSION, CONCAT(\'.\', OS_RELEASE)) AS VERSION FROM SYSIBMADM.ENV_SYS_INFO';
+  }
+
+  // Schema queries
+  createSchema(schema) {
+    return `CREATE SCHEMA "${schema}"`;
```

Comment:
This may fail if the schema includes a `"` character

and/or this may let users do things like `createSchema('db"."public')` which would result in `CREATE SCHEMA "db"."public"`

I think we should either escape ", or throw if it's present in the schema string.

This applied to other places where a schema name is inlined too

Reply:
I think on IBM i, schema names might only be able to be alphanumeric. Will look into this further.

Reply:
I tried to create a schema called `SCHE'MA"MAN` by running:
```
CREATE SCHEMA "SCHE'MA""MAN"
```
I received the following error:
```
Name "SCHE'MA""MAN" not allowed. Cause . . . . . :   "SCHE'MA""MAN" contains a character that is not allowed or does not contain any characters. An ordinary identifier must begin with (A-Z, $, #, or @) and be followed by zero or more (A-Z, 0-9, $, #, @, or _). A delimited identifier is a string of characters within SQL escape characters.  The characters allowed in delimited names depends on the type of name. -- The characters between the escape characters for system table names, schema names, package names and other system object names can be any characters except Hex 00-3F, Hex 40 (space), Hex 5C (*), Hex 6F (?), Hex 7D ('), Hex 7F ("), and Hex FF
```

So it appears that a valid schema should never include a `"` character, and if it does, I would expect it to crash. Do you think its best to let the DBMS handle the error, or try and throw our own error? My only fear is at some later date, the characters allowed in a valid schema might change, and in that case we'd have to fix it again.